scripts/docker/ubuntu20.04-mongod.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:20.04
```

Comment:
We really really need a docker registry, we have the AWS one, but i hate it. We could just retag the "official" mongo here: https://hub.docker.com/_/mongo, or just do as you did here, but build a set of mongo community and enterprise from each version of the server.

Reply:
Is that maybe something we could ask from the developer productivity team?

Reply:
Btw here you probably can start with "FROM mongo:4.4" or "FROM node:14", even combine both. You can try this for example (though i think it could miss some deps):

```
FROM mongo:4.4

COPY --from=node:14 /usr/local/bin/node /usr/local/bin/node
```

Reply:
> Is that maybe something we could ask from the developer productivity team?

Yeah they said the AWS registry is fine, even got one: https://eu-central-1.console.aws.amazon.com/ecr/repositories?region=eu-central-1, but in Okta i've seen a juicy artifactory repo with docker support enabled and configured with single sign-on ..  that would be so nice .. no idea why we can't use it, probably is more for production stuff 😕